charts/headlamp/README.md

```diff
@@ -70,8 +70,13 @@ $ helm install my-headlamp headlamp/headlamp --namespace kube-system
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | config.baseURL | string | `""` | base url path at which headlamp should run |
-| config.oidc.clientID | string | `""` | OIDC client ID |
-| config.oidc.clientSecret | string | `""` | OIDC client secret |
-| config.oidc.issuerURL | string | `""` | OIDC issuer URL |
-| config.oidc.scopes | string | `""` | OIDC scopes to be used |
 | config.pluginsDir | string | `"/headlamp/plugins"` | directory to look for plugins |
+
```

Comment:
For the sake of backward compatibility cant we have the values as `config.oidc.*` and use it as secrets instead of args?

Reply:
Makes sense. will do it